homeassistant/components/stream/worker.py

```diff
@@ -0,0 +1,141 @@
+"""Proides the worker thread needed for processing streams."""
+import asyncio
+from fractions import Fraction
+import io
+import logging
+
+from .const import AUDIO_SAMPLE_RATE
+from .core import Segment, StreamBuffer
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def generate_audio_frame():
+    """Generate a blank audio frame."""
+    from av import AudioFrame
+    audio_frame = AudioFrame(format='dbl', layout='mono', samples=1024)
+    audio_bytes = b''.join(b'\x00\x00\x00\x00\x00\x00\x00\x00'
```

Comment:
Wonder if this is faster:
```suggestion
    audio_bytes = b'\x00\x00\x00\x00\x00\x00\x00\x00' * 1024
```

Reply:
It's at least cleaner.  Learn something new every day 👍